module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUserMembership.md

```diff
@@ -63,7 +63,7 @@ This example demonstrates how to retrieve user memberships in Microsoft Entra ID
 
 ```powershell
 Connect-Entra -Scopes 'User.Read'
-Get-EntraBetaUserMembership  -ObjectId 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb' -All
+Get-EntraBetaUserMembership  -ObjectId 'SawyerM@contoso.com' -All
```

Comment:
nit: Both guid and `UserPrincipalName` work

Reply:
Yes. Both work but customers prefer UPN since it is more intuitive.